package.json

```diff
@@ -14,6 +14,9 @@
     "cover:check": "istanbul check-coverage && echo code coverage thresholds met, achievement unlocked!",
     "test:quick": "babel-node node_modules/.bin/_mocha -R tap test/init.js test/*-test.js"
   },
+  "pre-commit": [
+    "test"
```

Comment:
yeah we don't really need this, tests are run as part of CI here on github so we should be fine without it